ONSdigital/census-rm-sample-loader
# Generate Random Sample

# Motivation and Context
We need to be able to generate a realistic production-sized sample which we can use in our test environments.

# What has changed
Wrote a script to generate a random sample, based on production quantities of each treatment code.

# How to test?
Run `python generate_sample_file.py`. Manually check every single line of the contents of `sample_file.csv`.

# Links
Trello: https://trello.com/c/zJUzsWK0
